src/test/java/CartTests.java

```diff
@@ -0,0 +1,38 @@
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import shop.Cart;
+import shop.RealItem;
+
+public class CartTests {
+
+    Cart cart;
+    RealItem car;
+
+    @Before
```

Comment:
it is better to create new test data before each test. Please use here @BeforeEach annotation

Reply:
Fixed